protocol/0082-ETHD-ethereum-data-source.md

```diff
@@ -110,81 +110,79 @@ Select {
-4. Phase 2 - Create an oracle source listening for a particular event and specify an incorrect ABI format for the event. Proposal should fail validation and should throw an error (<a name="0082-ETHD-022" href="#0082-ETHD-022">0082-ETHD-022</a>)
-5. Create an oracle that calls a read method of a smart contract and specify an incorrect ABI format for the event. Proposal should fail validation and should throw an error (<a name="0082-ETHD-023" href="#0082-ETHD-023">0082-ETHD-023</a>)
-6. Will need some tests around consensus, will require setting up a network and having some nodes receive different values for the same oracle data point and testing that the oracle data point is/is not published depending on voting (<a name="0082-ETHD-024" href="#0082-ETHD-024">0082-ETHD-024</a>)
+1. Set up a new data source with invalid contract address - should fail validations (Phase 2 - listening for invalid event ) (<a name="0082-ETHD-030" href="#0082-ETHD-030">0082-ETHD-030</a>)
+2. Phase 2 - Set up a data source for listening to a particular event sent at a frequency of 2 secs. The oracle data source stops emitting events after emitting a couple of events. Raise and error via the TX RESULT event  if 5 consecutive events are missed - need to ratify / expand on this (<a name="0082-PLAZZO-005" href="#0082-PLAZZO-005">0082-PLAZZO-005</a>)
+3. Phase 2 - Create an oracle source listening for a particular event and specify an incorrect ABI format for the event. Proposal should fail validation and should return an error (<a name="0082-PLAZZO-006" href="#0082-PLAZZO-006">0082-PLAZZO-006</a>)
+4. Create an oracle that calls a read method of a smart contract and specify an incorrect ABI format for the event. Proposal should fail validation and should return an error (<a name="0082-ETHD-034" href="#0082-ETHD-034">0082-ETHD-034</a>)
+5. Will need some tests around consensus, will require setting up a network and having some nodes receive different values for the same oracle data point and testing that the oracle data point is/is not published depending on voting (<a name="0082-ETHD-035" href="#0082-ETHD-035">0082-ETHD-035</a>)
+6. Create 2 markets with exactly the same ethereum oracle data sources for termination and settlement. Verify that the data spec ID are the same and the data sources are ACTIVE. Now terminate and settlement Market1 using governance proposals. Now the status of the data sources should still be ACTIVE as Market2 is still using them. (<a name="0082-ETHD-036" href="#0082-ETHD-036">0082-ETHD-036</a>)
+7. Ensure that when a market is suspended and then resumed via a governance proposal we can still terminate and settle the market using ethereum oracle. (<a name="0082-ETHD-037" href="#0082-ETHD-037">0082-ETHD-037</a>)
```

Comment:
Can we move this to governance spec and move to the markest-by-governance part of the features.json file

Reply:
ACs 036 and 037 moved to governance spec, will raise a PR and reference here.